packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -371,18 +380,26 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
                       {t("test_preview")}
                     </Button>
                   </div>
+                  {!form.routes && (
```

Comment:
```suggestion
                  {!form.routes?.some((r) => !r.isFallback) && (
```

There is always a fallback route even if user didn't define any route, so we need to exclude that to identify if there is a user defined route or not.